packages/core/src/interpreter.ts

```diff
@@ -726,7 +729,7 @@ export class Interpreter<
 
       if (!child) {
         throw new Error(
-          `Unable to forward event '${event}' from interpreter '${this.id}' to nonexistant child '${id}'.`
+          `Unable to forward event '${event.name}' from interpreter '${this.id}' to nonexistant child '${id}'.`
```

Comment:
is it possible to trigger this error? forwarding happens before resolving transitions so `this.children` can't even be changed "in the middle" of the iteration as event queue is processed one by one. If my reasoning is correct then I would remove this, or leave it as dev-only with a piece of information that this is an internal error and it should be reported to us

Reply:
Yes, you can `forwardTo('nonExistantChild')` which will trigger the error.